src/sentry/tasks/post_process.py

```diff
@@ -844,6 +844,18 @@ def process_snoozes(job: PostProcessJob) -> None:
 
 
 def process_replay_link(job: PostProcessJob) -> None:
+    def _get_replay_id(event):
+        # replay ids can either come as a context, or a tag.
+        # right now they come as a context on non-js events,
+        # and javascript transaction (through DSC context)
+        # It comes as a tag on js errors.
+        # TODO: normalize this upstream in relay and javascript SDK. and eventually remove the tag
+        # logic.
+        context_replay_id = event.data.get("contexts", {}).get("replay", {}).get("replay_id")
+        tag_replay_id = event.get_tag("replayId")
+
+        return context_replay_id or tag_replay_id
```

Comment:
Not a big deal but if you replace `tag_replay_id` with the operation itself it will be lazily evaluated.